src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/CraftPriceTooltip.java

```diff
@@ -33,6 +33,7 @@ public CraftPriceTooltip(int priority) {
 
     @Override
     public void addToTooltip(@Nullable Slot focusedSloFt, ItemStack stack, List<Text> lines) {
+        if (SkyblockerConfigManager.get().general.itemTooltip.enableCraftingCost == GeneralConfig.Craft.OFF) return;
```

Comment:
this isn't necessary, call the isEnabled method before trying to add the tooltip